### PR TITLE
KEYCLOAK-13379 added now excluded project files

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

**Review by ssilvert — APPROVED** [review]

File: themes/src/main/resources/theme/keycloak-preview/account/resources/tsconfig.json
Comment (ASzc) [inline themes/src/main/resources/theme/keycloak-preview/account/resources/tsconfig.json]: A trailing comma isn't allowed in JSON:

```
aszczucz@tyrfing ~> curl -sSL https://raw.githubusercontent.com/keycloak/keycloak/695864d4b70494a0d24f7c54bf5a4cde2e907c45/themes/src/main/resources/theme/keycloak-preview/account/resources/tsconfig.json | jq '.'

parse error: Expected another array element at line 20, column 3
```

**Comment by vmuzikar** [comment]
This was merged rather fast. ;) Reviewing now, some tests failed, need to investigate if it's related...

**Comment by abstractj** [comment]
@vmuzikar I got the same impression, I was trying it locally and just realized it was merged.

@ssilvert for further interactions I'd suggest to wait for the approval from the original reporter.

File: themes/src/main/resources/theme/keycloak-preview/account/resources/tsconfig.json
Comment (vmuzikar) [reply themes/src/main/resources/theme/keycloak-preview/account/resources/tsconfig.json]: So what should we do now when it was already merged? Another follow-up JIRA?? @ssilvert

**Comment by sumitdas66** [comment]
image

I am getting the above issue when trying to load the account console when using 9.0.2 version. I thought there has been a fix for this on the mentioned version. But still I am getting the issue. 

Below is the exception:=

ERROR [org.keycloak.services.error.KeycloakErrorHandler] (default task-2) Uncaught server error: java.lang.NullPointerException
        at org.keycloak.keycloak-services@9.0.2//org.keycloak.services.resources.account.AccountFormService.init(AccountFormService.java:147)
        at org.keycloak.keycloak-services@9.0.2//org.keycloak.services.resources.account.AccountLoader.getAccountService(AccountLoader.java:86)
        at org.keycloak.keycloak-services@9.0.2//org.keycloak.services.resources.RealmsResource.getAccountService(RealmsResource.java:210)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)

**Pull request merged**
